### PR TITLE
Debug Viewer

### PR DESCRIPTION
**Pull request by xepozz**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ✔️
| Breaks BC?    | ❌
| Tests pass?   | ✔️

Added initialization of Debug Viewer

### CODE REVIEW

**Comment by samdark** [comment]
Merge after https://github.com/yiisoft/yii-debug-viewer/pull/3

**Comment by samdark** [comment]
[Yii debug](https://trello.com/c/r3CdJUaP/1-yii-debug)

File: public/index.php [hunk 1]
```diff
@@ -21,6 +21,8 @@
 if ($debugEnabled) {
```
Comment (yiiliveext) [inline public/index.php]: This change should be removed, the debugger starts automatically now.

**Comment by samdark** [comment]
Closing since https://github.com/yiisoft/yii-debug-viewer/pull/3 is closed.

**Pull request closed without merging**